commit 23ccfe892bb0e8bb2ac95ac88aa82ff2d3ba3411
Author: K. Y. Srinivasan <kys@microsoft.com>
Date:   Thu Dec 22 16:54:03 2016 -0800

    Drivers: hv: util: Backup: Fix a rescind processing issue
    
    commit d77044d142e960f7b5f814a91ecb8bcf86aa552c upstream.
    
    VSS may use a char device to support the communication between
    the user level daemon and the driver. When the VSS channel is rescinded
    we need to make sure that the char device is fully cleaned up before
    we can process a new VSS offer from the host. Implement this logic.
    
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1323_23ccfe892bb0e8bb2ac95ac88aa82ff2d3ba3411.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1323_23ccfe892bb0e8bb2ac95ac88aa82ff2d3ba3411.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
